src/components/shared/table/TractionTableRow.vue

```diff
@@ -0,0 +1,35 @@
+<!--/**
+   * # TractionTableRow
+   *
+   * * This component provides a wrapper for a boot-strap <b-tr> and HTML <tr> 
+   *
+   * @example
+   * 
+   */
+-->
+<template>
+  <flagged-feature name="enable_custom_table">
+    <template #disabled>
+      <b-tr>
+        <slot />
+      </b-tr>
+    </template>
+    <tr v-bind="$attrs" data-testid="row" :class="classes">
```

Comment:
Just checking. Do we show `<tr v-bind="$attrs" data-testid="row" :class="classes">` always used independent of custom table feature enabled or not? We have additional replacement when the feature is disabled.

Reply:
I doubt I understand the question properly.

'TractionTableRow' acts as a wrapper component which switches between html <tr> element and it's bootstrap equivalent <b-tr>. If we use traction-table-row with 'custom-table' feature flag  enabled, <tr> will be called otherwise <b-tr>. But  we are free to use <tr> directly anywhere as it as a normal html element. 

 If my answer doesn't make sense, please explain the question again.

Reply:
Sorry for not asking properly. Now I understand and see no issue.

Reply:
No worries, Happy to know that it is cleared 👍